CODE_OF_CONDUCT.md

```diff
@@ -23,23 +14,29 @@ All recipients of reports commit to:
     * **Node-RED:** <team@nodered.org>
     * **Webdriver.io:** [contact TSC reps](https://github.com/webdriverio/webdriverio/blob/HEAD/AUTHORS.md)
     * **Webhint:** <support@webhint.io>
-2. For every other OpenJS Foundation project, please email <report@lists.openjsf.org>. The Cross Project Council (CPC) is responsible for managing these reports.
+    
+    In case of a conflict of interest, if the report is not handled appropriately (for example, if there are unacceptable delays, a lack of communication, confidentiality isn't respected, etc.), or in case of an emergency, you may [escalate the report][escalation] to the [Code of Conduct Team (CoC Team)][CoC Team] by emailing <coc@openjsf.org> or by contacting any of its members.
+2.  **To report any other incident** (for example: in an OpenJS Foundation project not listed above, in a Collab Space, in the Cross Project Council, or during an event), please email the [Code of Conduct Team (CoC Team)][CoC Team] at <coc@openjsf.org> or contact any of its members.
+
+### Appeal a decision
```

Comment:
This section is weird, as it's mostly summarising content from `conduct/COC_POLICY.md#appeals`, but it's also defining from when the 30 days are counted, which isn't explicitly defined there.

Could this section be dropped, or replaced with a link to the policy's Appeals section? That would help deduplicate the text.

Also, the first "appeal" link doesn't work.

Reply:
Duplication is by design here, to offer a summary, in the CoC itself, for the benefit of reporters of their main course of action.

I think your below suggested fix addresses your other concern here.